dev/io.openliberty.org.jboss.logging35/bnd.bnd

```diff
@@ -21,7 +21,7 @@ Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=11))"
 Bundle-SymbolicName: io.openliberty.org.jboss.logging35; singleton:=true
 
 -includeresource: \
-  @${repo;org.jboss.logging:jboss-logging;3.5.0.Final;EXACT}
+  @${repo;org.jboss.logging:jboss-logging;3.5.0.Final;EXACT}!/!module-info.class
```

Comment:
jboss-logging also includes a `META-INF/INDEX.LIST` which includes the jar name and a list of directories it contains.

That file will now be wrong so I think we should also remove it, though I'm not sure if it will cause any problems.

We can probably state that any INDEX.LIST which comes from re-bundling a library will always be incorrect since we always change the file name.